packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -249,44 +253,91 @@ output teamsFxConfigurationOutput object = contains(reference(resourceId('Micros
-    expect(backupFolderItems.length).equals(1);
-    const parameterBackupFolder = path.join(
-      backupBaseFolder,
-      backupFolderItems[0],
-      parameterFolderName
-    );
-    const parameterBackupFiles = await fs.readdir(parameterBackupFolder);
-    expect(parameterBackupFiles.length).equals(1);
     expect(
-      await fs.readFile(path.join(parameterBackupFolder, parameterBackupFiles[0]), fileEncoding)
-    ).equals(mockedParameterDefaultJsonContent);
-    const templateBackupFolder = path.join(backupBaseFolder, backupFolderItems[0], templatesFolder);
-    const templateBackupFiles = await fs.readdir(templateBackupFolder);
-    expect(templateBackupFiles.length).equals(2);
-    expect(
-      await fs.readFile(path.join(templateBackupFolder, templateBackupFiles[0]), fileEncoding)
-    ).equals(mockedMainBicepContent);
+      await fs.readFile(path.join(projectArmParameterFolder, parameterFileName), fileEncoding)
+    ).equals(
+      `{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "provisionParameters": {
+      "value": {
+        "resourceBaseName": "mytestappdefa000000",
+        "FrontendParameter": "FrontendParameterValue",
+        "SimpleAuthParameter": "SimpleAuthParameterValue"
+      }
+    }
+  }
+}`
+    );
+    expect(await fs.pathExists(path.join(projectArmTemplateFolder, "../provision/bot.bicep"))).to.be
+      .false;
+    expect(await fs.pathExists(path.join(projectArmTemplateFolder, "../teamsFx/bot.bicep"))).to.be
+      .false;
+    selectedPlugins = [botPlugin];
+    mockedCtx.projectSettings = {
+      appName: testAppName,
+      projectId: uuid.v4(),
+      solutionSettings: {
+        hostType: HostTypeOptionAzure.id,
+        name: "azure",
+        version: "1.0",
+        activeResourcePlugins: [fehostPlugin.name, simpleAuthPlugin.name],
+        capabilities: [TabOptionItem.id, BotOptionItem.id],
+      },
+    };
+    result = await generateArmTemplate(mockedCtx, selectedPlugins);
+    expect(result.isOk()).to.be.true;
+    const fileContent = await fs.readFile(
+      path.join(projectArmParameterFolder, parameterFileName),
+      fileEncoding
+    );
+    expect(fileContent).equals(
+      `{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "provisionParameters": {
+      "value": {
+        "resourceBaseName": "mytestappdefa000000",
+        "BotParameter": "BotParameterValue",
+        "FrontendParameter": "FrontendParameterValue",
+        "SimpleAuthParameter": "SimpleAuthParameterValue"
+      }
+    }
+  }
+}`
+    );
+    expect(await fs.pathExists(path.join(projectArmTemplateFolder, "../provision/bot.bicep"))).to.be
+      .true;
+    expect(await fs.pathExists(path.join(projectArmTemplateFolder, "../teamsFx/bot.bicep"))).to.be
+      .true;
```

Comment:
we may need to check the existing files do not change in the test case

Reply:
sure